lib/ast/expression2.ts

```diff
@@ -171,13 +172,15 @@ export class FunctionCallExpression extends Expression {
             } else if (this.name === 'attimer') {
                 const time = this.in_params.find((ip) => ip.name === 'time')!;
                 const expiration_date = this.in_params.find((ip) => ip.name === 'expiration_date');
+                const once = new BooleanValue(false);
```

Comment:
This is not correct. You should look for the parameter called "once" in the list of parameters to the "attimer" function (aka `this.in_params`)